pkg/services/ngalert/models/alert_rule.go

```diff
@@ -240,6 +240,8 @@ type AlertRule struct {
 	PanelID         *int64  `xorm:"panel_id"`
 	RuleGroup       string
 	RuleGroupIndex  int `xorm:"rule_group_idx"`
+	Record          string
+	RecordFrom      string
```

Comment:
There _might_ be another field called `RecordTo`, indicating the datasource to write the results to. Whether we want a per-rule datasource, or just config.ini knobs for that, is still up for debate in the design doc and has not reached consensus. So, I've left that field out for now.

That's where the current name came from. "RecordFrom" and "RecordTo" make a nice pair, if both exist - but open to alternative suggestions.

Reply:
In the new struct format we could call this `To` or `Target`. Less important now though